erdjs/src/stats.ts

```diff
@@ -0,0 +1,91 @@
+import { IApiProvider } from "./interface";
+
+/**
+ * An object holding Network stake parameters.
+ */
+export class Stats {
+    private static default: Stats;
+
+    /**
+     * The number of Shards.
+     */
+    public Shards: number;
+
+    /**
+     * The Number of Blocks.
+     */
+    public Blocks: number;
+    /**
+     * The Number of Accounts.
+     */
+    public Accounts: number;
+    /**
+     * The Number of transactions.
+     */
+    public Transactions: number;
+    /**
+     * The Refresh rate.
+     */
+    public RefreshRate: number;
+    /**
+     * The Number of the current Epoch.
+     */
+    public Epoch: number;
+    /**
+     * The Number of rounds passed.
+     */
+    public RoundsPassed: number;
+    /**
+     * The Number of Rounds per epoch.
+     */
+    public RoundsPerEpoch: number;
+
+    constructor() {
+        this.Shards = 0;
+        this.Blocks = 0;
+        this.Accounts = 0;
+        this.Transactions = 0;
+        this.RefreshRate = 0;
+        this.Epoch = 0;
+        this.RoundsPassed = 0;
+        this.RoundsPerEpoch = 0;
+    }
+
+    /**
+     * Gets the default configuration object (think of the Singleton pattern).
+     */
+    static getDefault(): Stats {
```

Comment:
The `NetworkConfig` object is used when creating `Transaction` objects, thus we added this `getDefault()`. Do we need this for `Stats` as well?

Reply:
Nope, I just removed the getDefault Function